lib/solr_wrapper/instance.rb

```diff
@@ -41,9 +45,10 @@ def wrap(&_block)
     ##
     # Start Solr and wait for it to become available
     def start
-      extract
+      raise_error_unless_extracted
```

Comment:
Can you say something about this change? I feel like it's a feature that things were automatically extracted as-needed.